staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -213,3 +224,48 @@ func (f *FieldManager) buildManagerInfo(prefix string, operation metav1.ManagedF
 	}
 	return internal.DecodeManager(&managerInfo)
 }
+
+// StripFields that don't contribute much value to an objects managedFields
+func (f *FieldManager) StripFields(obj runtime.Object, typed typed.TypedValue, apiVersion fieldpath.APIVersion, managed fieldpath.ManagedFields, manager string) error {
+	set, err := typed.ToFieldSet()
+	if err != nil {
+		return fmt.Errorf("failed to get fieldset: %v", err)
+	}
+
+	strippedFields := &unstructured.Unstructured{
```

Comment:
This might be simplified by using [NewSet](https://github.com/kubernetes-sigs/structured-merge-diff/blob/master/fieldpath/set.go#L36) and [MakePath](https://github.com/kubernetes-sigs/structured-merge-diff/blob/master/fieldpath/path.go#L45), and then you wouldn't need to specify fake values like `"strip"` and `time.Now().UTC().Format(time.RFC3339)`, and you wouldn't need to do any conversion to get the field set to diff with.

Something like:
```
stripSet := fieldpath.NewSet()
stripSet.Insert(fieldpath.MakePath("metadata", "name"))
stripSet.Insert(fieldpath.MakePath("metadata", "namespace"))
...
```

Reply:
Oops didn't see Antoine's comment